app/assets/javascripts/mumuki_laboratory/application/button.js

```diff
@@ -0,0 +1,37 @@
+class Button {
```

Comment:
The `Button` class may collide with other `Button` components. I'd recommend to create it within a proper namespace